src/lib/legacy.ts

```diff
@@ -6712,15 +6728,16 @@ const toZigbee2 = {
         convertSet: async (entity, key, value: any, meta) => {
             if (['_TZE200_5toc8efa'].includes(meta.device.manufacturerName)) {
                 await sendDataPointValue(entity, dataPoints.moesHeatingSetpoint, value * 10);
-            } else {
-                if (value <= 5) value = Math.round(value*10);
-                await sendDataPointValue(entity, dataPoints.moesHeatingSetpoint, value);
```

Comment:
This changes the behaviour for all except `_TZE200_5toc8efa`, please revert this change.